lib/Service/ConfigService.php

```diff
@@ -65,7 +65,7 @@ public function getAll(): array {
 
 	public function get($key) {
 		$result = null;
-		[$scope, $id] = explode(':', $key, 2);
+		[$scope] = explode(':', $key, 2);
```

Comment:
```suggestion
		[$scope] = explode(':', $key, 1);
```

if you don't care about the rest :)

Reply:
No, that would basically not explode the value at all:

> If limit is set and positive, **the returned array will contain a maximum of limit elements with the last element containing the rest of string**.

````
$foo = explode(':', 'a:b:c:d', 1);
array(1) {
  [0]=>
  string(7) "a:b:c:d"
}
````